lib/child_process.js

```diff
@@ -622,13 +611,8 @@ function validateMaxBuffer(maxBuffer) {
 
 function validateKillSignal(killSignal) {
   if (typeof killSignal === 'string' || typeof killSignal === 'number') {
-    killSignal = lookupSignal(killSignal);
-
-    if (killSignal === 0)
```

Comment:
Prior to this, zero could be passed to `ChildProcess#kill()`, but not as the kill signal in `execFile()` and company. That seems like something we might want to maintain.